src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,104 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int START_CAPACITY = 10;
+    private static final double COEFFICIENT_MULTIPLY = 1.5;
+    private int size;
+    private Object[] array = new Object[START_CAPACITY];
```

Comment:
let's initialize all non constant fields in constructor

Reply:
ok